php/util/class-fieldmanager-util-term-meta.php

```diff
@@ -1,15 +1,19 @@
 <?php
 /**
- * Use fieldmanager to store meta data for taxonomy terms.
+ * Class file for Fieldmanager_Util_Term_Meta.
  *
- * This is deprecated as of WordPress 4.4, which introduces Term Meta into core.
+ * @package Fieldmanager
+ */
+
+/**
+ * Use Fieldmanager to store meta data for taxonomy terms.
+ *
+ * This is deprecated as of WordPress 4.4, which introduces term MetA into core.
  * If you were using this feature prior to 4.4, it will continue to operate
  * until probably WordPress 4.8, however it is in your best interest to migrate
```

Comment:
That's correct. That was just my estimation for when we'll officially deprecate it.